docs-examples/example-kotlin/build.gradle

```
@@ -1,12 +1,13 @@
 plugins {
     id 'io.micronaut.build.internal.nats-examples'
-    id("org.jetbrains.kotlin.jvm")
-    id("org.jetbrains.kotlin.kapt")
-    id("org.jetbrains.kotlin.plugin.allopen")
+    id("io.micronaut.build.internal.kotlin-kapt")
     id("io.micronaut.build.internal.kotlin-base")
 }
 
 micronaut {
     testRuntime "kotest5"
 }
 
+dependencies {
+    implementation(projects.testSuiteNatsUtils)

```

Should not this be in the test classpath? 

```suggestion
    testImplementation(projects.testSuiteNatsUtils)
```

---

https://github.com/micronaut-projects/micronaut-nats/pull/652/commits/bfd5e2bbd6358c2c83ee018f7178033e1819cb3c